spec/requests/v6/assessments_controller/realistic_civil_apply_spec.rb

```diff
@@ -0,0 +1,244 @@
+                    "date": "2023-04-01",
+                    "amount": 200.0,
+                    "client_id": "3000-01-01",
+                  },
+                  {
+                    "date": "2023-05-01",
+                    "amount": 200.0,
+                    "client_id": "3000-01-01",
+                  },
+                  {
+                    "date": "2023-06-01",
+                    "amount": 160.0,
+                    "client_id": "3000-01-01",
+                  },
+                ],
+              },
+            ],
+          },
+          "dependants": [
+            {
+              "assets_value": 0.0,
+              "relationship": "child_relative",
+              "date_of_birth": "2020-08-14",
+              "monthly_income": 0.0,
+              "in_full_time_education": true,
+            },
+            {
+              "assets_value": 0.0,
+              "relationship": "child_relative",
+              "date_of_birth": "2021-08-14",
+              "monthly_income": 0.0,
+              "in_full_time_education": false,
+            },
+          ],
+          "employment_income": [
+            {
+              "name": "Job 1",
+              "payments": [
+                {
+                  "tax": -159.6,
+                  "date": "2023-02-20",
+                  "gross": 1868.25,
+                  "client_id": "3000-01-01",
+                  "benefits_in_kind": 0.0,
+                  "national_insurance": -98.43,
+                  "net_employment_income": 1610.22,
+                },
+              ],
+              "client_id": "3000-01-01",
+            },
+          ],
+          "irregular_incomes": {
+            "payments": [
+              {
+                "amount": 8000.0,
+                "frequency": "annual",
+                "income_type": "student_loan",
+              },
+            ],
+          },
+          "other_incomes": [
+            {
+              "source": "Maintenance in",
+              "payments": [
+                {
+                  "date": "2023-04-26",
+                  "amount": 167.0,
+                  "client_id": "3000-01-01",
+                },
+              ],
+            },
+          ],
+          "outgoings": [
+            {
+              "name": "rent_or_mortgage",
+              "payments": [
+                {
+                  "amount": 563.08,
+                  "client_id": "3000-01-01",
+                  "payment_date": "2023-05-02",
+                  "housing_cost_type": "mortgage",
+                },
+              ],
+            },
+            {
+              "name": "maintenance_out",
+              "payments": [
+                {
+                  "amount": 30.0,
+                  "client_id": "3000-01-01",
+                  "payment_date": "2023-05-15",
+                },
+              ],
+            },
+          ],
+          "properties": {
+            "main_home": {
+              "value": 720_000.0,
```

Comment:
The original submission doesn't have underscores - there are a few of these that have crept in

Reply:
So in Ruby those underscores are purely cosmetic - 720000.0 and 720_000.0 are both floating point representations of 720,000.

Reply:
ah of course, this is a Ruby hash, not actually JSON. Would prefer it to be without the underscore still though, unless rubocop is going to complain

Reply:
rubocop doesnt like it

Reply:
👍